vault/expiration.go

```diff
@@ -185,7 +193,7 @@ func (m *ExpirationManager) Revoke(leaseID string) error {
 	}
 
 	// Revoke the entry
-	if err := m.revokeEntry(le); err != nil {
+	if err := m.revokeEntry(le); err != nil && !force {
 		return err
```

Comment:
I would at least log the error in `force` mode instead of silently swallowing it